### ISSUE TITLE
 When you restart Home Assistant, the Xplora integration takes a long time to start

### ISSUE THREAD

**Issue by MaggiMaikel** [Ludy87/xplora_watch#261]
### What version of Integration has the issue?

V 2.10.1

### System Health details

Raspberry 4

### Checklist

- [X] I have enabled debug logging for my installation.
- [x] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).

### Describe the issue

Hello everyone,

First of all, thank you for your work, I think it's great that I can integrate my daughter's smartwatch into HA.

Now to my problem:

Yesterday evening I updated the Xplora integration. Maybe I should have read beforehand that all the entities change. I have installed the smartwatch in many automations. It was definitely fun to change everything up 😀

In any case, since updating to the latest version, the Xplora integration takes almost exactly 5 minutes to restart with a home assistant. That wasn't the case before and unfortunately I can't use the version because I can't and don't want to wait 5 minutes for integration every time I restart.

I have already completely reinstalled and set up the integration, but unfortunately the problem still persists. Is the error known and could you perhaps help me?

Best regards

Additional: I come from Germany and this text was translated using Google Translate.

### Reproduction steps

1.
2.
3.
...
Nothing

### Debug logs

```text
Nothing
```

**Comment by Ludy87** [comment]
Hallo @MaggiMaikel ich werde schauen, ob ich was machen kann. Kannst du mir sagen, welches Intervall du eingestellt hast?

**Comment by MaggiMaikel** [comment]
Hello,

I left the interval at default (180 seconds).

**Comment by hoebart** [comment]
Any progress in this?
It's getting a bit annoying when you need to restart Home Assistant since other integrations are waiting to start up.

**Comment by Ludy87** [comment]
@MaggiMaikel  I have two clocks that are initializing and it takes 26 seconds

![image](https://github.com/Ludy87/xplora_watch/assets/4592558/873a1488-02a1-47d5-a346-c11f16a1cdcf)

**Comment by MaggiMaikel** [comment]
Hello,

I tried it today with the latest version. Unfortunately the same problem still exists, which is why I unfortunately have to install a backup.

**Comment by Ludy87** [comment]
Try: https://github.com/Ludy87/xplora_watch/pull/348

**Comment by MaggiMaikel** [comment]
OK, great, thank you very much. 

I'll try it in the next few days as soon as I have time. I hope it works better then. I will let you know and thank you again for your work.

**Comment by MaggiMaikel** [comment]
Hello, 

I couldn't wait to install the update. But unfortunately the problem still exists. But I was able to narrow it down further. This is school mode. Start xplora_watch.switch... This is what slows down the reboot very much. I also noticed that even if the reboot is finished at some point, the school mode switch shows up as unavailable. Unfortunately, I will have to install a backup again because school mode plays an important role for us and we can't do without it.
